### ISSUE TITLE
mapping terms with different syntax for values

### ISSUE THREAD

**Issue by wdduncan** [mapping-commons/SSSOM#52]
For the DWC-MIXS mappings (see [repo](https://github.com/tdwg/gbwg/tree/main/dwc-mixs)), we are finding terms that map in sense that they are about the same thing, but the specification for the syntax of the values is different. An example of this is depth. In DWC, the unit and the scalar are in separate fields, but in MIxS the scalar and unit are in one field (e.g. "3 meters"). See this ticket:

https://github.com/tdwg/gbwg/issues/28

Do we want to develop mappings between to capture differences in the value syntax? FWIW, I think `DWC:depth` should be mapped as a `skos:exactMatch` to `MIXS:depth`. The differing value syntax is an ETL/implementation issue. However, it might still be nice to have a vocabulary for capturing such differences.

cc @cmungall @raissameyer @pbuttigieg

**Comment by matentzn** [comment]
Interesting!

**Comment by matentzn** [comment]
@wdduncan do you think we should solve this on SSSOM level? If so, any suggestion how? While I absolutely see the value, I am sceptical about how to do this, other than providing a new sub-hierarchy of annotation properties..

**Comment by wdduncan** [comment]
@matentzn I share your skepticism, which is why I added my comment:

> FWIW, I think DWC:depth should be mapped as a skos:exactMatch to MIXS:depth. The differing value syntax is an ETL/implementation issue. However, it might still be nice to have a vocabulary for capturing such differences.

Matching syntaxes could be another standard unto itself :)

**Comment by ddooley** [comment]
Just a note that in OBI a few of us pushed for and got a data property [has representation](http://purl.obolibrary.org/obo/OBI_0002815) that enables one to hold these syntactically encoded strings, as a precursor to actually parsing them into parts.

**Comment by wdduncan** [comment]
@matentzn I don't think this is necessary out of scope: mapping has both a semantic and syntactic dimension. For example, the DarwinCore to MIxS working group is capturing syntactic differences by adding **syntax predicate** column (see file [here](https://docs.google.com/spreadsheets/d/1k6Xe1OREUVISLjw1XLrtLqWsE7QgvWf7lSIXEbqXDpA/edit#gid=0)).

**Comment by mellybelly** [comment]
shouldn't 'has representation' go in IAO or RO?

**Comment by linikujp** [comment]
Thumbs up for this issue. I am watching to see if a solution will be agreed
here. There are a lot of this kind of examples in data elements mapping,
such as a person's age in year, or in days. Might be off the topic here...
but I am interested in following up with this.

Thanks,
Asiyah

**Comment by ddooley** [comment]
@mellybelly I'm open to that idea of moving the data property to IAO or RO. It would be neat if RO took on ALL OBOFoundry relations. When we introduced it I recall not having the impression that IAO relations were being developed, but since that time IAO is being curated by OBI/IAO team jointly (i.e. same telecons).

**Comment by matentzn** [comment]
We should discuss this in person @wdduncan once. Its again too complex for a GitHub issue discussion. Maybe we can use a pattern using UO in conjunction with #43 to solve this problem as well;

**Comment by ddooley** [comment]
This is analogous to exact semantic match on IAO "data item" but different measurement system syntax that comes forth in various ontologies via 'has quantity', 'has unit', 'has representation', 'has value specification' etc.

**Comment by matentzn** [comment]
Still a bit queasy about this feature, but there seems to be some pressure. I think the idea of this ticket is now fully subsumed by @cmungall #61 if no one complains until the workshop, I will close this issue here in favour of the other. Basically #61 proposes to define a function that need to be applied to instances of a class/field prior to interpreting the matching predicate.

**Comment by matentzn** [comment]
To capture that two things are essentially about the same thing, but the representation is using different serialisations (units, languages etc) is probably entirely beyond SSSOM now - it would just make it very complicated, at least if you also wanted to somehow encode the _way in which the serialisations are different_. If this is just an ask for a new mapping relation "semapv:exactMatchDifferentSerialisation" this would probably also be rejected but maybe worth a try on the SEMPAV repo.
